openstreetmap-carto.style

```diff
@@ -21,14 +20,10 @@ node       capital      text         linear
 node,way   construction text         linear
 node,way   covered      text         linear
 node,way   culvert      text         linear
-node,way   cutting      text         linear
 node,way   denomination text         linear
-node,way   disused      text         linear
 node       ele          text         linear
-node,way   embankment   text         linear
 node,way   foot         text         linear
 node,way   generator:source    text  linear
-node,way   harbour      text         polygon
```

Comment:
The same is true for any other tag that indicates an area and will not have a column